GCRCatalogs/register.py

```diff
@@ -7,11 +7,11 @@
 import requests
 import socket
 from GCR import BaseGenericCatalog
-from .utils import is_string_like
+from .utils import is_string_like, get_config_dir
 
 __all__ = [
-    "get_root_dir", "set_root_dir", "reset_root_dir", "get_available_catalogs",
-    "get_reader_list", "get_catalog_config", "has_catalog", "load_catalog", "retrieve_paths", "get_site_list", "set_root_dir_by_site"]
+    "get_root_dir", "set_root_dir", "remove_root_dir_default", "reset_root_dir", "get_available_catalogs",
+    "get_reader_list", "get_catalog_config", "has_catalog", "load_catalog", "retrieve_paths", "get_site_list", "set_root_dir_by_site", "write_to_user_config", "remove_from_user_config", "remove_root_dir_default"]
```

Comment:
`remove_root_dir_default` repeated above.
```suggestion
    "get_reader_list", "get_catalog_config", "has_catalog", "load_catalog", "retrieve_paths", "get_site_list", "set_root_dir_by_site", "write_to_user_config", "remove_from_user_config"]
```